package.json

```diff
@@ -18,14 +18,18 @@
     "extract-translations": "turbo extract-translations -- --config ../../tools/i18next-parser.config.js"
   },
   "dependencies": {
+    "@hookform/resolvers": "^3.3.1",
```

Comment:
While this isn't terribly important, it would be preferrable, I think, to keep all dependencies in `devDependencies`. These aren't in any way different from how we handle, e.g., `dayjs`.

Reply:
Thanks @ibacher !  I thought I was just keeping this consistent with the patient chart, but it looks like they are all devDependencies there, I will correct.

Reply:
Yeah we are somewhat inconsistent on this because we changed the norm a couple times during the first couple years of O3. I don't think there is a technical difference between `devDependencies` and `dependencies` for us, with our current tooling and setup, though at least some monorepo tools install `devDependencies` at the monorepo level and `dependencies` at the package level—but I think Yarn 4 just treats them the same.

Oddly, my inclination would be the opposite of what @ibacher suggests here—I am inclined to follow the rule that a dependency that gets built/bundled goes in `dependencies`. But I am quite open to being convinced otherwise :shrug: